R/process_data.R

```diff
@@ -66,13 +69,18 @@ process_pacta_results <- function(data, start_year, end_year, time_horizon,
       check_company_ticker_mapping()
   }
 
+  data_processed %>%
```

Comment:
turns out the fail when running snapshot test for loans is because column company_id is NA for loans. one could either create a placeholder company_id for loans or disregard that column in report_missings() for loans